unittest/CMakeLists.txt

```
@@ -115,6 +116,10 @@ add_python_unit_test("py-std-array" "unittest/python/test_std_array.py"
                      "python;unittest")
 set_tests_properties("py-std-array" PROPERTIES DEPENDS ${PYWRAP})
 
+add_python_unit_test("py-std-pair" "unittest/python/test_std_pair.py"
+                     "python;unittest")
```

```suggestion
                     "unittest")
```
Since we don't import eigenpy, we don't need python module in PYTHONPATH.